### ISSUE TITLE
Review mentions of OneGeology Europe

### ISSUE THREAD

**Issue by bgsmase** [OneGeology/documentation#21]
Apart from #20 there are some other references to OneGeology Europe. Consider whether they need removing as the project no longer exists or whether they can remain as historical references or arbitrary examples.

In the ContactView feature table (https://onegeology.github.io/documentation/providingdata.html#contactview-features) under the contactType_uri it says there is no INSPIRE vocabulary but mentions what OneGeology Europe project used. Should it just say use the CGI vocabulary and remove any reference to what OneGeology Europe did?

In the ShearDisplacementView and GeologicUnitView feature tables (https://onegeology.github.io/documentation/providingdata.html#sheardisplacementstructureview-features and https://onegeology.github.io/documentation/providingdata.html#geologicunitview-features) below the representative...Age_uri rows, OneGeology Europe is mentioned as it was the source of additional PreCambrian Epoch age terms. Should double check the CGI vocabularies are still split in two like this and consider whether to retain reference to OneGeology Europe. Double check the INSPIRE vocabulary includes those extra terms. Maybe rephrase to say that the ics ones are those found in the official ICS chart and the other ones were added because of requirements of OneGeology Europe and are included in INSPIRE vocabulary?

At the end of the Core TC211/ISO:19115 Metadata section of the OneGeology profile (https://onegeology.github.io/documentation/providingdata.html#core-tc211-iso-19115-metadata) OneGeology Europe participants are requested to note extra requirements of INSPIRE metadata. Probably this should be changed to refer to any services that want / have to conform to INSPIRE requirements.

The MapServer documentation at https://onegeology.github.io/documentation/providingdata.html#handling-fields-for-which-you-have-no-data mentions OneGeology Europe not requiring some data that GeoSciML-Lite does need. Perhaps this should be rephrased just as general advice for anyone who doesn't have data for some GeoSciML-Lite required field?

The example MapServer configuration snippets in https://onegeology.github.io/documentation/providingdata.html#configuring-group-layering have abtracts referring to the data coming from OneGeology Europe work. These are harmless and could be left as arbitrary example abstract text for a service.

In the Apache reverse proxy section (https://onegeology.github.io/documentation/providingdata.html#using-apache-http-server-as-a-reverse-proxy) there is an example of reverse proxying the BRGM 1GConnector. This is just an arbitrary example in some ways but probably a bit confusing as an out-of date reference? Just remove it?

**Comment by nmtoken** [comment]
> At the end of the Core TC211/ISO:19115 Metadata section of the OneGeology profile (https://onegeology.github.io/documentation/providingdata.html#core-tc211-iso-19115-metadata) OneGeology Europe participants are requested to note extra requirements of INSPIRE metadata. Probably this should be changed to refer to any services that want / have to conform to INSPIRE requirements.

replaced with:

> Note, conformance of metadata to the ISO 19115 Core does not guarantee conformance to INSPIRE metadata, see the INSPIRE technical guidelines document `MD_IR_and_ISO_v1_2_20100616 <http://inspire.ec.europa.eu/documents/Metadata/MD_IR_and_ISO_20131029.pdf>`_ for further details.
